deb-arm/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM ${BASE_IMAGE}
 
 # Build Args
 ARG GIMME_GO_VERSION=1.15.11
-ARG DD_CONDA_VERSION=4.7.12-0
+ARG DD_CONDA_VERSION=4.8.3-0
```

Comment:
Why this specific version instead of the last one (4.10.1)?

Reply:
Because mistakes were made, let me change it to the latest.

Reply:
Okay looks like 4.10 says 

> NOTE: This release formally drops support for Python 2.7 and Python < 3.6.

but we can use the last 4.9.x

Reply:
I checked here and the latest one listed is 4.8: https://docs.conda.io/en/latest/miniconda_hashes.html

Where did you get your version number from?

Reply:
Ok, there's a 4.10 version for conda-forge: https://github.com/conda-forge/miniforge/releases/
But 4.9 is the latests for miniconda: https://repo.anaconda.com/miniconda/

Reply:
I've updated the PR to use the latest of each, and moved the version number definition to the `setup_python.sh` script so we can change it from a single place. 

This adds a potential difference between the arm and x64 builders, but on the other hand if conda-forge released 4.10 ahead of miniconda it might be because of a bug in 4.9, so I think this is better.

Reply:
Oh, I didn't see your comment about 4.10 🤦